tests/cockpit-tasks-centosci.yaml

```diff
@@ -0,0 +1,83 @@
+---
+apiVersion: v1
+kind: List
+items:
+- kind: ReplicationController
+  metadata:
+    name: cockpit-tasks
+  spec:
+    replicas: 1
+    selector:
+      infra: cockpit-tasks
+    template:
+      metadata:
+        labels:
+          infra: cockpit-tasks
+      spec:
+        containers:
+        - name: cockpit-tasks
```

Comment:
Can we please name this "centosci-tasks", and rename the -bos one "bos-tasks"? This makes it possible to see which cloud they were running on, and where to log in for "oc rsh". I'm not entirely sure that it's *this* `name:` which determines that, it could also be the `ReplicationController` name (but I think that's unlikely).